File: Assignment_2/Assignment2_5.py
"""-------------------------------------------------------------------------------------------
                          Assignment2_5
                    (Student name - Vaishali Jorwekar)
                     Python by Marvellous Infosystems
---------------------------------------------------------------------------------------------
Problem statement : This program accepets a number from user and 
                    checks if given number is prime or not
------------------------------------------------------------------------------------------"""
#----------------------------------------------------------------------------------------
# This function checks if given number is prime or not
#---------------------------------------------------------------------------------------

def checkPrime(number):
    isPrime = True   
    for num in range(2,number):
         if (number % num == 0) :
                isPrime = False
                break
    return isPrime      
#------------------------------------------------------------------------------------------
# This function accepts a input number from user and calls checkPrime(number) function
#------------------------------------------------------------------------------------------
def main():
    number=int(input("Enter number :"))

    if number==0 or number==1:
        print(f"\n{number} is not Prime Number")
    else:
        isPrimeNumber = checkPrime(number)
        if isPrimeNumber:
             print(f"\n{number} is Prime Number")
        else:
             print(f"\n{number} is not Prime Number")          
#-----------------------------------------------------------------------------------
# This is main entry point of the program
#-----------------------------------------------------------------------------------
if __name__=="__main__":
    main()
#-----------------------------------------------------------------------------------

File: JavaPractice/Assignment_2/Assignment2_5.java

/*-------------------------------------------------------------------------------------------
                           Assignment2_5
                    (Student name - Vaishali Jorwekar)
                     Python by Marvellous Infosystems
---------------------------------------------------------------------------------------------
Problem statement : This program accepts a number from user and 
                    checks if number is PRIME
------------------------------------------------------------------------------------------*/
import java.util.Scanner;




public class Assignment2_5 {
	
	public static boolean checkPrime(int number) {
		boolean isPrime=true;
		if(number<=1) {
			isPrime= false;
		}
		for(int cnt=2;cnt<number;cnt++) {
			if(number%cnt==0) {
				isPrime= false;
				break;
			}
		}
		return isPrime;
	}

	public static void main(String[] args) {
		Scanner inputObj=new Scanner(System.in);
		System.out.println("Enter the number:");
		int number=inputObj.nextInt();
		boolean flag=checkPrime(number);
		System.out.println(flag);
		if(flag)
			System.out.println(number+" is a PRIME number");
		else
			System.out.println(number+" is not a PRIME Number");
			
		inputObj.close();
		

	}

}
